_episodes/07-find.md

```diff
@@ -374,7 +374,7 @@ Miscellaneous:
 > the commands `grep` and `wc` and a `|`, while another might utilize
 > `grep` options.
 > There is often more than one way to solve a programming task, so a
-> particular solution is usually chosen based on a combination of
```

Comment:
Similar to particular above, this is to highlight the singular solution that is chosen.

Reply:
Addressed in the next commit